pkg/handler/aws/participation/participation.go

```diff
@@ -0,0 +1,101 @@
+package main
```

Comment:
i think naming this file to main.go, make all Lamda functions consistent. (other AWS handlers files are main.go)

Reply:
There is another file in this package named main.go